CHANGES.md

```diff
@@ -1,10 +1,14 @@
 # Change Log
 
-#### next release (8.3.2)
+#### next release (8.3.3)
 
 - Fixed a bug when restoring timefilter from a share link having more than one imagery item with the same base URL (but different layer names).
 - [The next improvement]
 
+#### 8.3.2
+
+- Fix WPS duplicate display of analysis results when loaded through a share URL
+
```

Comment:
I think the correct next release for this change is `8.3.2` (so after the timefilter bug fix).

https://github.com/TerriaJS/terriajs/blob/15d468656bee43b0f872075cbf867475993e2536/CHANGES.md?plain=1#L3-L7